src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTypeCheck.java

```diff
@@ -40,7 +40,7 @@
 
 /**
  * <p>
- * Checks Javadoc comments for class and interface definitions. By default, does not check
+ * Checks the Javadoc of a type. By default, does not check
```

Comment:
To me I liked the usage of class/interface over type. I don't see anywhere else it says what type means for those that don't make the connection.

Reply:
This should be `annotation, enum, class, interface`. Too verbose, in my opinion.

Reply:
@pbludov Do you have another suggestion? or you wish to leave it as it is?

Reply:
changed to `Checks the Javadoc comments for annotation/enum/class/interface definitions.`